src/script/sign.h

```diff
@@ -22,13 +22,32 @@ struct CMutableTransaction;
 
 struct KeyOriginInfo
 {
-    unsigned char fingerprint[4];
+    unsigned char fingerprint[4]; // First 32 bits of the Hash160 of the public key at the root of the path
```

Comment:
`//!` please so that it shows up in doxygen

Reply:
Done